node/cn/filters/filter.go

```diff
@@ -159,6 +163,14 @@ func (f *Filter) indexedLogs(ctx context.Context, end uint64) ([]*types.Log, err
 	// Iterate over the matches until exhausted or context closed
 	var logs []*types.Log
 
+	maxItems := math.MaxInt32 - 1
+	if val := ctx.Value(getLogsCxtKeyMaxItems); val != nil {
+		val, ok := val.(int)
+		if ok {
+			maxItems = val
+		}
```

Comment:
Don't we need to return error for this case?

Reply:
Also, as the same code is used in line#214~line#220, how about using a function for this?

Reply:
I added a function for the duplicated code.

Reply:
@ehnuje 
1. `nil` value for `getLogsCxtKeyMaxItems` is allowed since `indexedLogs` can be called without the context key.
2. We can consider returning an error for the type assertion failure even though the code cannot fail without improper code fixes. However, users always return errors if improper code fixes happened. I think using `MaxInt32-1` is better than stopping `klay_getLogs` API serving.

Reply:
I considered the second case, to notify the user that he/she entered the wrong parameter. 
BTW, what does "even though the code cannot fail without improper code fixes" mean?

Reply:
The value of `getLogsCxtKeyMaxItems` is set by the init flag `api.filter.getLogs.maxitems`. If the value is given with an invalid type, Klaytn node will fail to start returning an parse error.

Reply:
Oh, I see. Then we don't need to worry about it!